Makefile

```diff
@@ -344,6 +344,6 @@ x-build-dist: export X_CARGO_CMD=build
 x-build-dist: export X_CARGO_FEATURES=${ENABLE_FEATURES}
 x-build-dist: export X_CARGO_RELEASE=1
 x-build-dist: export X_CARGO_CONFIG_FILE=${DIST_CONFIG}
-x-build-dist: export X_PACKAGE=cmd
+x-build-dist: export X_PACKAGE=tikv-server tikv-ctl
```

Comment:
Should it be quoted?

Reply:
No.

At first I quoted it. But it turned out the quote would be included in `X_PACKAGE`.

Reply:
But I don't see tikv-ctl in the graph above.

Reply:
Only `tikv-server` is compiled in the comparison graph. I use the following commands.
Before:
```
cargo build --release --package cmd --bin tikv-server --no-default-features --features "jemalloc mem-profiling portable sse protobuf-codec" -Ztimings
```
After:
```
cargo build --release --package tikv-server --no-default-features --features "jemalloc mem-profiling portable sse protobuf-codec" -Ztimings
```

`make release` or `make dist_release` will compile both `tikv-server` and `tikv-ctl`.